ExecutionHelper/.env.example

```diff
@@ -0,0 +1 @@
+WALLET_PRIVATE_KEY=
```

Comment:
```suggestion
# This is one of the rich accounts from the `era-test-node`.
WALLET_PRIVATE_KEY="0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110"
```

Reply:
We will need to have the same public key in the repo's root directory to deploy the whole thing.

I have a couple ideas:
- [ ] Move this to the root directory and symlink or copy it to `./executionHelper/.env`
- [ ] Use direnv, which propagates to all the children directories.
- [ ] See if dotenv supports a config to look for `../.env` instead of `.env`

Reply:
We can pass a --private-key flag to the compiler, so how about as part of a make target?

Reply:
Makefile targets don't take parameters, so it would be the same